### PR TITLE
feat(resize): add native resize capabilities to editor

### PR DESCRIPTION
**Pull request by emoralesb05**
instead of using automaticLayout + layout fixes

This to improve performance and allow proper e2e testing since the intervals would get in the way.

Also removed unwanted scrollbars and resize in a single step instead of adjustments every 200ms.

### CODE REVIEW

**Review by christianmemije — CHANGES_REQUESTED** [review]
Let's not pass in automaticLayout to the monaco editor and just use this implementation instead.

**Review by christianmemije — APPROVED** [review]
LGTM

File: src/platform/code-editor/code-editor.component.scss [hunk 1]
```diff
@@ -1,7 +1,12 @@
 :host {
-    display:block;
+  display: block;
+  position: relative;
+  overflow: hidden;
```
Comment (emoralesb05) [inline src/platform/code-editor/code-editor.component.scss]: Reminder to remove this.

File: src/platform/code-editor/code-editor.component.ts [hunk 2]
```diff
@@ -52,6 +57,12 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
 
   @ViewChild('editorContainer') _editorContainer: ElementRef;
 
+  /**
+   * automaticLayout?: boolean
+   * @deprecated in favor of our own resize implementation.
+   */
+  @Input('automaticLayout') automaticLayout: boolean = true;
```
Comment (emoralesb05) [inline src/platform/code-editor/code-editor.component.ts]: Reminder to add a console.warn for deprecation

**Pull request merged**